src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -89,72 +99,66 @@ public static void worldTeleport(Player entity, ResourceKey<Level> planet, doubl
 
-        if (item0 && item1 && item2 && item3) {
-            return true;
-        }
-        return false;
+        return true;
     }
 
-    public static boolean spaceSuitCheckBoth(LivingEntity entity) {
-        Boolean item3 = checkArmor(entity, 3, ModInit.OXYGEN_MASK.get());
-        Boolean item2 = checkArmor(entity, 2, ModInit.SPACE_SUIT.get());
-        Boolean item1 = checkArmor(entity, 1, ModInit.SPACE_PANTS.get());
-        Boolean item0 = checkArmor(entity, 0, ModInit.SPACE_BOOTS.get());
-
-        Boolean item3_2 = checkArmor(entity, 3, ModInit.NETHERITE_OXYGEN_MASK.get());
-        Boolean item2_2 = checkArmor(entity, 2, ModInit.NETHERITE_SPACE_SUIT.get());
-        Boolean item1_2 = checkArmor(entity, 1, ModInit.NETHERITE_SPACE_PANTS.get());
-        Boolean item0_2 = checkArmor(entity, 0, ModInit.NETHERITE_SPACE_BOOTS.get());
-
-        Boolean check3 = false;
-        Boolean check2 = false;
-        Boolean check1 = false;
-        Boolean check0 = false;
-
-        if (item3 || item3_2) {
-            check3 = true;
-        }
-        if (item2 || item2_2) {
-            check2 = true;
+    /**
+     * @param entity the LivingEntity to check
+     * @return true if entity is wearing a full set of standard or netherite space gear, false otherwise
+     */
+    public static boolean spaceSuitCheckBoth(final LivingEntity entity) {
+        if (!(checkArmor(entity, 3, ModInit.OXYGEN_MASK.get()) || checkArmor(entity, 3, ModInit.NETHERITE_OXYGEN_MASK.get()))) {
+            return false; // missing an oxygen mask
         }
-        if (item1 || item1_2) {
-            check1 = true;
+
+        if (!(checkArmor(entity, 2, ModInit.SPACE_SUIT.get()) || checkArmor(entity, 2, ModInit.NETHERITE_SPACE_SUIT.get()))) {
+            return false; // missing a space suit
         }
-        if (item0 || item0_2) {
-            check0 = true;
+
+        if (!(checkArmor(entity, 1, ModInit.SPACE_PANTS.get()) || checkArmor(entity, 1, ModInit.NETHERITE_SPACE_PANTS.get()))) {
+            return false; // missing space pants
         }
 
-        if (check0 && check1 && check2 && check3) {
-            return true;
+        if (!(checkArmor(entity, 0, ModInit.SPACE_BOOTS.get()) || checkArmor(entity, 0, ModInit.NETHERITE_SPACE_BOOTS.get()))) {
+            return false; // missing space boots
         }
 
-        return false;
+        return true; // has mask + suit + pants + boots
     }
 
-    public static boolean checkArmor(LivingEntity entity,int number, Item item) {
+    public static boolean checkArmor(final LivingEntity entity, final int number, final Item item) {
         return entity.getItemBySlot(EquipmentSlot.byTypeAndIndex(EquipmentSlot.Type.ARMOR, number)).getItem() == item;
     }
 
     public static boolean isSpaceWorld(Level world) {
-        if (Methods.isWorld(world, moon)
+        if (Methods.isWorld(world, overworld)) {
+            return false;
+        }
```

Comment:
Avoid checking against every space world if they're in the overworld. This saves 10 checks when in the overworld.